### ISSUE TITLE
English language error

### ISSUE THREAD

**Issue by travillaintim** [JulienNigon/SpellcasterUniversity-Bug-Report#1002]
Not a bug - just a language error. See picture attached
![English typo](https://user-images.githubusercontent.com/3208897/79082810-16d04f80-7cef-11ea-8ed8-a0dc2f8fc0a6.png)

**Comment by Neelan22** [comment]
Yes this is a problem in English. It's a model sentence used for all futures, which is not very good in English. We need to think about how to fix that.
Thank you !
